schemas/context/segmentmembership.schema.json

```diff
@@ -0,0 +1,60 @@
+{
+  "meta:license": [
+    "Copyright 2018 Adobe Systems Incorporated. All rights reserved.",
+    "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$id": "https://ns.adobe.com/xdm/context/segmentMembership",
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "title": "Segment Membership",
+  "type": "object",
+  "description": "Details about a segment membership.",
+  "definitions": {
+    "segmentMembership": {
+      "properties": {
+        "xdm:segmentID": {
+          "title": "Segment ID",
+          "$ref": "https://ns.adobe.com/xdm/context/identity",
+          "description": "The identity of the segment or snapshot definition in with the domain of the specific system that processes that type of segment."
+        },
+        "xdm:version": {
```

Comment:
@kstreeter I think so. Have we thought about doing version more broadly on entities across anything stored in XDM? I kind of feel strongly we should be storing id + version anywhere we record any event or action. It will make debugging customer issues much easier.

Reply:
Yes it is an opaque string.